src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -175,11 +175,8 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
         return isAllowed;
     }
 
-    // TODO: this will be included in GATK after Barclay is updated, because now it have this method
-    // TODO: https://github.com/broadinstitute/barclay/blob/976fafea23216cf0577b8e3fe635b115c7975a76/src/main/java/org/broadinstitute/barclay/argparser/CommandLinePluginDescriptor.java#L164
-    // TODO: although there is a change in the signature, because I require that this is a list over T
-    // @Override
-    public List<ReadFilter> getDefaultInstances() {
+    @Override
+    public List<Object> getDefaultInstances() {
```

Comment:
Add "// TODO: the signature should be change in the future" including pointer to Barclay issue/PR